src/anemoi/training/config/training/default.yaml

```diff
@@ -68,6 +68,15 @@ validation_metrics:
     # other kwargs
     ignore_nans: True
 
+# List of validation metrics to keep in normalised space, and scalars to apply
+# Cannot be used for metrics associated with variables that are remapped between the
+# real space and internal model space.
+scale_validation_metrics:
+  scalars_to_apply: ['variable']
+  metrics:
+    - 'all'
```

Comment:
If adding 'all' here is the new default, the runs break when applying the remapper since the number of variables does not match the number of scalars. Do we need an assert/warning that prevents from calling the weighted validation loss in the case the remapper is applied?

Reply:
Addressed in [f51fbb1](https://github.com/ecmwf/anemoi-training/pull/202/commits/f51fbb13ecd13f80aa5477f503a1c42d5aef5846)